content/blog/2019/06/2019-06-06-cloudfeature-gsoc-cp1.adoc

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "GSoC Project Progress : Cloud features for external workspace manager"
+tags:
+- jenkins
+- gsoc
+author: yufei_zhang
+description: "Short summary of coding period 1" # optional
+opengraph:
+---
+
+=== About Me
+My name is Yufei Zhang, a student from Zhejiang University and Simon Fraser University Duel Degree Program. I want to share the discoveries when we are exploring the possiblities of extending current external workspace manager plugin (referred to as EWM) to support AWS EFS. 
```

Comment:
Dual Degree Program, not Duel. Very different :-).

Reply:
On it : ) I am a fighter